Sprint Planning Docs/Sprint Planning#2.md

```diff
@@ -0,0 +1,54 @@
+# Sprint Planning #2
+
+**Trello Board**: https://trello.com/b/Do4WCnFy/teachers-pet-🐢
+
+# Project Summary
+
+Teacher's Pet is an app aimed at streamlining the office hours process. The goal of this app is to help instructors best help their students during office hours with ease in an effective, efficient manner. 
+
+# Tasks Completed
+
+**Ashley Valdez**
+- Got 1 TA To Test Our App 
+    - Bhavye Jain, CS TA @ UCSB.
+- Thinking of talking to Butner about having learning assistants test Teacher’s Pet.
+- [Contributed Frameworks] (https://www.notion.so/ashleys-workspace/Milestone-1-f9ac176336c24f648876866ccb21ecd0) that Teacher's Pet may require.
+
+
+**Toniya Patil**
+- [Completed the hi-fi Figma design] (https://www.notion.so/ashleys-workspace/Milestone-1-f9ac176336c24f648876866ccb21ecd0) for Teacher's Pet.
```

Comment:
In the future, for anything related to design that was created in Figma, you can link the Figma file. As an example, the hi-fi Figma designs could have been linked here